cmake_modules/DeployVST3.cmake

```diff
@@ -51,6 +50,7 @@ endfunction()
 
 function(fixup_vst3_plugin_bundle bundle dirs)
 
+  include(BundleUtilities)
```

Comment:
BundleUtilities is only ever supposed to be included as part of an install target and prints out a warning otherwise, so was making the CMake output noisy. Probably shouldn't have snuck this into the PR but figured as I was messing with CMake anyway..